### ISSUE TITLE
OJDBC artifect error when building dgva-server with maven

### ISSUE THREAD

**Issue by AdrianBZG** [EBIvariation/eva-ws#81]
When building eva-ws using 'maven clean install', it's giving an error building dgva-server, giving "Authorization Required" error on ORACLE maven servers for OJDBC, using NoTravis profile:

![image](https://user-images.githubusercontent.com/8275330/36755846-b6d21d1e-1c0d-11e8-9a2f-0843428a51cb.png)

OS: Ubuntu 16 x64

If it's not an error, and I'm just not configuring it correctly, sorry, I'm new to the EVA repositories!

**Comment by AdrianBZG** [comment]
Please check pull request #82, it fixes the issue, building correctly and passing all the tests. I changed the OJDBC repo to use [1], same version of the previous one, which doesn't give the "Authorization Required" error.

Thanks.

[1] https://mvnrepository.com/artifact/com.github.noraui/ojdbc8/12.2.0.1

**Comment by jmmut** [comment]
TL;DR: you have 2 options: download the driver in the oracle's way, or try to exclude the dgva submodule if you are not interested in that part.

In any case, thanks for contributing, but I'm afraid we can not merge the PR because that artifact seems to point to a private github repo, relicensing with BSD, which sounds strange to say the least.

## Option 1: The oracle's way
As much as we disagree with oracle license ideology, we are using them at the moment and we have to be legal, and use the official repositories and binaries.

The oracle license states things like:
> [...] You shall require Your end users to execute an agreement binding them to terms [...]

Because of that, they require us to tell every user of us to download manually the binaries, accepting the license, or register and use a key to be able to download it programmatically. Here you have some useful links, like the register page and "Technical information about how to use the Oracle Maven Repository" http://www.oracle.com/webfolder/application/maven/index.html

Honestly, our documentation on this should be improved, but we didn't expect much developer users of this part of EVA.

## Option 2: Exclude it!
We haven't really tried this, but maybe you can use the maven parameter `-pl '!dgva-server'` to exclude that submodule if you just care about the main EVA webservices. 
https://stackoverflow.com/questions/5539348/how-to-exclude-a-module-from-a-maven-reactor-build/22783674#22783674

**Comment by AdrianBZG** [comment]
Thanks for the info!
